projects.html

```diff
@@ -2,8 +2,273 @@
 <html>
 <head>
 	<title>Projects</title>
+	<style>
+     .img1{
+      width: 650px;
+	  height: 500px;
+      top: 20px;
+	  position: relative;
+	  z-index: 1;
+	  left:25px;
+	  border: 3px outset white;
+      border-radius: 50px;
+	  opacity: 0.9;
+      
+
+	 }
+	 
+	 .img2{
+      width: 600px;
+      height: 450px;
+	  left: 50px;
+      bottom: 50px;
+	  position:relative;
+      z-index: 1;
+	  border: 3px outset white;
+      border-radius: 40px;
+	  opacity: 0.9;
+	 }
+	 .img3{
+      width: 600px;
+      height: 500px;
+	  position: relative;
+      z-index: 1;
+	  left: 20px;
+	  border: 3px outset white;
+      border-radius: 30px;
+	  opacity: 0.6;
+
+	 }
+	 .img4{
+      width: 600px;
+	  height: 550px;
+      bottom: 30px;
+	  left: 100px;
+	  position: relative;
+      z-index: 1;
+      border: 3px outset white;
+      border-radius: 30px;
+	  opacity: 0.9;
+	 }
+	 .img5{
+      width: 800px;
+	  height: 400px;
+      top: 40px;
+	  left: 350px;
+	  position: relative;
+      z-index: 1;
+	  border: 3px outset white;
+      border-radius: 70px;
+	  opacity: 0.9;
+	 }
+      
+	 
+.button{
+    padding: 8px 14px;
+    cursor: pointer;
+    font-size: 15px;
+    display: inline-block;
+    
+      text-align: center;
+    letter-spacing: 0.2125rem;
+    overflow: hidden;
+    background: rgba(255,255,255,.1);
+    position: relative;
+    text-align: center;
+    text-transform: uppercase;
+    border-radius: 30px;
+    z-index: 2;
+    box-shadow: 0 0 10px rgba(0,0,0.2);
+    
+    transition: 
+        background 5s cubic-bezier(0.19, 1, 0.22, 1),
+        border 1s cubic-bezier(0.19, 1, 0.22, 1),
+        color 0.6s cubic-bezier(0.19, 1, 0.22, 1);
+    user-select: none;
+  }
+  
+  .button a {
+    color: white;
+    font-family: "Varela Round";
+    text-decoration: none;
+  }
+  
+  .button:hover {
+    background-color: rgba(255, 255, 255, 0.05);
+    border-color: #fff;
+    box-shadow: 0 0 8px rgba(255, 245, 245, 0.8);
+    transition: background 0s;
+  }
+  .button:hover a {
+    color: #fff;
+  }
+   
+  .button1{
+	right: 27%;
```

Comment:
check the indentation, it should one tab space only

Reply:
👍